collect_app/src/main/java/org/odk/collect/android/dao/FormsDao.java

```diff
@@ -44,6 +44,13 @@ public Cursor getFormsCursor(String selection, String[] selectionArgs) {
         return getFormsCursor(null, selection, selectionArgs, null);
     }
 
+    public int deleteFormsForFormId(String formId) {
```

Comment:
Thanks. I like the coding style of the project, and wouldn't want to disturb it in my patches.